genomics_quality_control/genomics_quality_control.md

```diff
@@ -0,0 +1,927 @@
+
+The second part starts with a `:` and then gives the absolute path
+of the files you want to transfer from your remote computer. Don't
+forget the `:`. We used a wildcard (`*.html`) to indicate that we want all of
+the HTML files.
+
+The third part of the command gives the absolute path of the location
+you want to put the files. This is on your local computer and is the
+directory we just created `~/Desktop/fastqc_html`.
+
+<div class = "help">
+<b style="color: rgb(var(--color-highlight));">Troubleshooting help</b><br>
+
+**Note on using zsh**:
+If you are using zsh instead of bash (macOS for example changed the default recently to zsh), it is likely that a no matches found error will be displayed.
+The reason for this is that the wildcard ("*") is not correctly interpreted. To fix this problem the wildcard needs to be escaped with a "\":
+
+```bash
+scp dcuser@ec2-34-238-162-94.compute-1.amazonaws.com:~/genomics_tools_and_methods/results/fastqc_untrimmed_reads/\*.html ~/Desktop/fastqc_html
+```
+
+Alternatively, you can put the whole path into quotation marks:
+
+```bash
+scp "dcuser@ec2-34-238-162-94.compute-1.amazonaws.com:~/genomics_tools_and_methods/results/fastqc_untrimmed_reads/*.html" ~/Desktop/fastqc_html
+```
+
+</div>
+
+You should see a status output like this:
+
+``` +output
+SRR2584863_1_fastqc.html                      100%  249KB 152.3KB/s   00:01    
+SRR2584863_2_fastqc.html                      100%  254KB 219.8KB/s   00:01    
+SRR2584866_1_fastqc.html                      100%  254KB 271.8KB/s   00:00    
+SRR2584866_2_fastqc.html                      100%  251KB 252.8KB/s   00:00    
+SRR2589044_1_fastqc.html                      100%  249KB 370.1KB/s   00:00    
+SRR2589044_2_fastqc.html                      100%  251KB 592.2KB/s   00:00
+```
+
+Now we can go to our new directory and open the 6 HTML files.
+
+Depending on your system, you should be able to select and open them all at once via a right click menu in your file browser.
+
+### Quiz: FastQC
+
+Take a look at your samples. Which sample(s) looks the best in terms of per base sequence quality? Which sample(s) look the worst?
+
+[[describe sample quality]]
+<script>
+  let input = "@input".trim();
+  /.*/i.test(input);
+</script>
+***
+<div class = "answer">
+
+(Note that we can't automatically grade an open-ended question like this, so it will be marked "correct" no matter what you write.)
+
+All of the reads contain usable data, but the quality decreases toward the end of the reads.
+
+</div>
+***
+
+## Decoding the other FastQC outputs
+
+We have now looked at quite a few "Per base sequence quality" FastQC graphs, but there are nine other graphs that we have not talked about!
+Below we have provided a brief overview of interpretations for each of these plots.
+For more information, please see the FastQC documentation [here](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/)
```

Comment:
```suggestion
For more information, please see the FastQC documentation [here](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
```

Reply:
Thanks! I'll also update this link to make it informative link text.